### ISSUE TITLE
A11y_Visual Studio Code for MAC_Azure Function_Create New Project_Keyboard: The user is unable to access the tooltip info using keyboard when keyboard focus lands on 'Create new project' button.

### ISSUE THREAD

**Issue by bbarik06** [microsoft/vscode-azurefunctions#3156]
GitHubTags: #A11yMAS;#A11yTCS;#A11ySev2;#BM_BM_AzureToolsforVSCodeMAC_May2022 ;# Azure Tools for VS Code;#MAC;#FTP;#DesktopApp;#Voiceover;#WCAG2.1.1

![1_A11y_Visual Studio Code for MAC_Azure Function_Create New Project_Keyboard](https://user-images.githubusercontent.com/98393966/167086939-8bef556c-fe45-4bd6-9256-06d0582f9737.gif)

## Environment Details:

Application Name: Visual Studio Code
Application Version: 1.66.2 (Universal)
Mac Version: macOS Monterey 12.3.1

Screen Reader: Voiceover

## Repro Steps:

1. Launch Visual Studio Code.
2. Click on extension button and search for "Azure Functions".
3. Click on install and tab to the "Azure" button.
4. Tab till 'Functions' button and hit enter.
5. Tab till 'Create new project' button.
6. Use keyboard to observe the tooltip info.

## Actual:
The user is unable to access the tooltip info using keyboard when keyboard focus lands on 'Create new project' button.

The issue can be observed throughout the application.

## Expected:
The user should be able to access the tooltip info using keyboard when keyboard focus lands on 'Create new project' button.

## User Impact:
Keyboard users will lose out important tooltip information.

**Comment by MicroFish91** [comment]
This should be covered by this issue: https://github.com/microsoft/vscode/issues/52016.

This is a special case that only affects users who are using keyboard and are not using a screen reader. This comment sums up this case quite well: https://github.com/microsoft/vscode/issues/52016#issuecomment-641096443

Basically, VS Code team has not received any user requests to add a custom hover feature and it is not supported by modern web browsers, so there is no plan to pursue this at the current time.  Thank you.

**Comment by fsteffi** [comment]
@MicroFish91 this duplicate bug https://github.com/microsoft/vscode/issues/52016. is opened in windows - win11 this bug is for MAC

The issue addressed in the above bug is for native controls or known controls like close min max

the issue in this bug is referred to controls that are not unique, so the bug https://github.com/microsoft/vscode/issues/52016. cant be linked to this

**Comment by MicroFish91** [comment]
> @MicroFish91 this duplicate bug [microsoft/vscode#52016](https://github.com/microsoft/vscode/issues/52016). is opened in windows - win11 this bug is for MAC

My interpretation of isidorn's comment would be that his response was OS agnostic.  He seems to be entertaining the idea of whether or not this should be pursued and has come to the conclusion that it doesn't really make sense to pursue this.  MacOS doesn't seem like a key point in this context.

> The issue addressed in the above bug is for native controls or known controls like close min max
> the issue in this bug is referred to controls that are not unique, so the bug [microsoft/vscode#52016](https://github.com/microsoft/vscode/issues/52016). cant be linked to this

I'm not sure I understand, neither the linked issue nor this issue refer to a unique tooltip.  The examples used ('Create function, 'Deploy to function app', 'refresh', 'Collapse All') are used to demonstrate that tooltips are not shown when hovering over with keyboard.  This is extremely similar to the linked issue, so I'm not sure why the determination would not apply here as well.

In either case, this is not something that can be fixed at the extension level.  Thank you!

**Comment by fsteffi** [comment]
From the user perspective we already read the aria label of the element once the user navigates on it. Aria label is usually the same as what is hovered or better, so this issue is covered for blind users.
The only users this makes sense for imho are users who can not use the mouse and are not using a screen reader. Thus they navigate using the keyboard to an action and they would like to know more details about that action. So it would be nice to have an explicit command to trigger this native hover for those users. However web browsers do not support this as far as I know and it is not possible to trigger a native hover.

We can maybe look into this once MS Teams starts supporting this.
The only solution I see is to build a custom hover which I would not do for a feature which none of the users requested.

comment from external team

**Comment by bbarik06** [comment]
@MicroFish91 Please join office hour call for further clarification: https://outlook.office365.com/owa/calendar/microsoft2779007@microsoft.onmicrosoft.com/bookings/s/liYlILD8qUK6LE24eXV-Vg2

**Comment by MicroFish91** [comment]
Informational Note - Upstream issue, open for tracking purposes: https://github.com/microsoft/vscode/issues/132344

**Comment by MicroFish91** [comment]
Hi @bbarik06 I believe the upstream issue has been resolved, can you verify and close issue if satisfactory?  Thank you!

**Comment by Mana-swi** [comment]
#a11ydb-hover-only-content;
